include/wil/wistd_functional.h

```diff
@@ -516,9 +516,16 @@ namespace wistd     // ("Windows Implementation" std)
     swap_wil(function<_Rp(_ArgTypes...)>& __x, function<_Rp(_ArgTypes...)>& __y)
     {return __x.swap(__y);}
 
+    // std::invoke

```

Comment:
Please copy the implementation from libc++

Reply:
The libc++ implementation uses invoke_result_t, which requires C++14.

Reply:
Feel free to reduce that limitation as there's no requirement that `invoke_result` be limited to C++17, apart from that's when it was introduced to the standard. Could also use `__invoke_of`

Reply:
Fixed